tests/computed.test.tsx

```diff
@@ -32,7 +25,7 @@ const sleep = (ms: number) =>
 
 describe('proxyWithComputed', () => {
   it('simple computed getters', async () => {
-    const computeDouble = jest.fn((x: number) => x * 2)
+    const computeDouble = vi.fn((x) => x * 2)
```

Comment:
```suggestion
    const computeDouble = vi.fn((x: number) => x * 2)
```

Reply:
@Aslemammad Please do the same for others.

Reply:
sure.